estela-api/core/tasks.py

```diff
@@ -37,6 +47,11 @@ def get_default_token(job):
 
 @celery_app.task
 def run_spider_jobs():
+    """
+    Task to run spider jobs that are in the IN_QUEUE_STATUS.
+    Selects jobs from the queue and updates their status to WAITING_STATUS.
+    Initializes job execution using the job manager.
+    """
```

Comment:
@joaquingx You are right. The task is not actively used since the request to launch jobs is usually sent without an async parameter. However, the logic is present here in case jobs are launched with the async parameter here: https://github.com/bitmakerla/estela/blob/5da817d67161eb060dd401f4b7757ab0d6b5aa4e/estela-api/api/views/job.py#L147-L153